lib/taggers/vcc_tagger.rb

```diff
@@ -0,0 +1,40 @@
+require_relative '../writing'
+
+class VCCTagger
+
+  def initialize(project)
+    @project = project
+  end
+
+  def create_tags
+    @vcc_tag = Tag.find_or_create_by!(
+        name: 'VCC',
+        shortname: 'vcc',
+        color: '#c686d9',
+        icon: 'description',
+        description: Writing.tag_article('vcc')
+    )
+  end
+
+  def apply_tags
+    Vulnerability.where(project: @project).each do |v|
+      if has_vcc?(v)  # vcc present
+        VulnerabilityTag.create!(
+            vulnerability: v,
+            tag: @vcc_tag,
```

Comment:
```suggestion
            tag: @vcc_tag,
            importance: 0.25,
```
This has low importance compared to other vulnerability-tags, so let's arbitrarily give it this. (The importance number is used to size and order things in the UI)